pywr/_recorders.pyx

```diff
@@ -1,20 +1,44 @@
 import numpy as np
 cimport numpy as np
-
 from pywr._core cimport Timestep
-
 import pandas as pd
+from past.builtins import basestring
 
 recorder_registry = set()
 
+cdef enum AggFuncs:
```

Comment:
I think I hit this trying to do this ...

https://github.com/cython/cython/issues/531

https://groups.google.com/forum/#!msg/cython-users/ZoLsLHwnUY4/74_JZaZvAAAJ